docs/integrations/web-servers/opentelemetry/apache-tomcat-opentelemetry.md

```diff
@@ -0,0 +1,276 @@
+---
+id: apache-tomcat-opentelemetry
+title: Apache Tomcat - OpenTelemetry Collector
+sidebar_label: Apache Tomcat - OTel Collector
+description: Learn about the Sumo Logic OpenTelemetry App for Apache Tomcat.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<img src={useBaseUrl('img/integrations/web-servers/apache-tomcat.png')} alt="Thumbnail icon" width="65"/> <img src={useBaseUrl('img/send-data/otel-color.svg')} alt="Thumbnail icon" width="45"/>
+
+The [Apache Tomcat](https://tomcat.apache.org/tomcat-8.5-doc/index.html) app is a logs app that helps you get insight into visitor locations, traffic patterns, errors, resource utilization, garbage collection, web server operations and access from known malicious sources.
+
+Tomcat logs are sent to Sumo Logic through Opentelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver).
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Tomcat-OpenTelemetry/Apache-Tomcat-Schematics.png')} alt="Schematics" />
+
+## Fields Created in Sumo Logic for Tomcat
+
+Following are the [Fields](https://help.sumologic.com/docs/manage/fields/) which will be created as part of Tomcat App install if not already present.
+
+- **`webengine.cluster.name`** - User configured. Set a value of the cluster where your tomcat instance resides. This will be tagged along with the data sent to sumo.
```

Comment:
```suggestion
- **`webengine.cluster.name`** - User configured. Set a value of the cluster where your Tomcat instance resides. This will be tagged along with the data sent to Sumo.
```